core/evaluation.py

```
@@ -0,0 +1,396 @@
+        if "conversation" in result:
+            for msg in reversed(result["conversation"]):
+                if msg.get("role") == "assistant":
+                    content = msg.get("content", "")
+                    # 提取 <response> 标签内容
+                    match = re.search(r"<response>(.*?)</response>", content, re.DOTALL)
+                    if match:
+                        return match.group(1).strip()
+                    return content
+
+        # 尝试从 spec 文件读取
+        if "spec_file" in result and result["spec_file"]:
```

Ruff (0.15.10)

[warning] Unnecessary key check before dictionary access

Replace with `dict.get`

(RUF019)